build.gradle

```diff
@@ -92,7 +90,7 @@ subprojects {
 	}
 
 	dependencyManagement {
-		imports { mavenBom "org.springframework.boot:spring-boot-starter-parent:${springBootVersion}" //mavenBom "org.springframework.cloud:spring-cloud-starter-parent:${springCloudStarterParentTrain}"
+		imports { mavenBom "org.springframework.boot:spring-boot-starter-parent:${springCloudStarterParentTrain}"
```

Comment:
Also added switching gradle-all back to gradle-bin

On Mon, Nov 30, 2015 at 6:44 AM, Corby Page <cpage@pivotal.io> wrote:

> Yeah, that's not right, thanks. I've added the fix for that to the pull
> request.
>
> On Mon, Nov 30, 2015 at 6:21 AM, David Ferreira Pinto <
> notifications@github.com> wrote:
>
>> In build.gradle
>> <https://github.com/dpinto-pivotal/cf-SpringBootTrader/pull/43#discussion_r46136701>
>> :
>>
>> > @@ -92,7 +90,7 @@ subprojects {
>> >  	}
>> >
>> >  	dependencyManagement {
>> > -		imports { mavenBom "org.springframework.boot:spring-boot-starter-parent:${springBootVersion}" //mavenBom "org.springframework.cloud:spring-cloud-starter-parent:${springCloudStarterParentTrain}"
>> > +		imports { mavenBom "org.springframework.boot:spring-boot-starter-parent:${springCloudStarterParentTrain}"
>>
>> I can't find a
>> org.springframework.boot:spring-boot-starter-parent:Brixton.M3 in any repos.
>>
>> note the boot rather than cloud
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/dpinto-pivotal/cf-SpringBootTrader/pull/43/files#r46136701>
>> .
>>
>
>
>
> --
> Corby Page
> Senior Field Engineer, Pivotal
> 713.584.3168
>
>


-- 
Corby Page
Senior Field Engineer, Pivotal
713.584.3168